src/script/conversation/ConversationService.ts

```diff
@@ -430,4 +430,17 @@ export class ConversationService {
   async mlsGroupExistsLocally(groupId: string): Promise<boolean> {
     return this.coreConversationService.mlsGroupExistsLocally(groupId);
   }
+
+  /**
+   * Will check if mls group is established locally.
+   * Group is established after the first commit was sent in the group and epoch number is at least 1.
+   * @param groupId groupId of the conversation
+   */
+  async isMLSGroupEstablishedLocally(groupId: string): Promise<boolean> {
+    return this.coreConversationService.isMLSGroupEstablishedLocally(groupId);
+  }
+
+  async getMLS1to1Conversation(userId: QualifiedId) {
+    return this.apiClient.api.conversation.getMLS1to1Conversation(userId);
```

Comment:
don't we have a `core` method that would do that? As much as possible we should avoid using the `apiClient` directly in the webapp (in the long run the apiClient should be hidden to the webapp)

Reply:
We do not have it yet but i will add it 👌. I like the approach you're suggesting 🙏